v23.1/backup.md

```diff
@@ -310,6 +310,36 @@ job_id             |  status   | fraction_completed | rows | index_entries | byt
 (1 row)
 ~~~
 
+### Restrict nodes executing a backup job by locality
```

Comment:
Thoughts on the title here? "Restrict" may be negative? Perhaps, "control" or something similar.